config/kubernetes-sigs/org.yaml

```diff
@@ -176,6 +176,7 @@ members:
 - interma
 - irfanurrehman
 - janetkuo
+- jayunit100
 - javier-b-perez
```

Comment:
The entry should be after `javier-b-perez` 👍

Reply:
Haha ok will fix